v4/test/testsuit/generics.go

```diff
@@ -44,6 +54,10 @@ func CreateWeaviateTestSchemaFood(t *testing.T, client *weaviate.Client) {
 	assert.Nil(t, propErrT2)
 	propErrA2 := client.Schema().PropertyCreator().WithClassName("Soup").WithProperty(descriptionProperty).Do(context.Background())
 	assert.Nil(t, propErrA2)
+	propErrT3 := client.Schema().PropertyCreator().WithClassName("Pizza").WithProperty(priceProperty).Do(context.Background())
+	assert.Nil(t, propErrT3)
+	propErrA3 := client.Schema().PropertyCreator().WithClassName("Soup").WithProperty(priceProperty).Do(context.Background())
```

Comment:
What do `T3`, `A3` stand for?

Reply:
It was Things and Actions, it's a leftover o times before v1